packages/start/src/client/tsrScript.ts

```diff
@@ -49,7 +49,10 @@ const __TSR__: StartTSRGlobal = {
             let controller
             ex.value = new ReadableStream({
               start(c) {
-                controller = c
+                controller = {
+                  enqueue: (chunk: unknown) => { try { c.enqueue(chunk); } catch {} },
+                  close: () => { try { c.close(); } catch {} },
+                }
```

Comment:
Doing this here seems to be the place that is the most defensive - anywhere "further out", it could swallow errors e.g. if controller is still undefined etc.